File: app/src/main/java/com/ajdi/yassin/popularmoviespart1/data/model/TrailersResponse.java
package com.ajdi.yassin.popularmoviespart1.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * @author Yassin Ajdi.
 */
public class TrailersResponse {

    @SerializedName("results")
    private List<Trailer> trailers;

    public List<Trailer> getTrailers() {
        return trailers;
    }

    public void setTrailers(List<Trailer> trailers) {
        this.trailers = trailers;
    }
}

File: app/src/main/java/com/ajdi/yassin/popularmovies/data/local/model/TrailersResponse.kt
package com.ajdi.yassin.popularmovies.data.local.model

import com.google.gson.annotations.SerializedName

/**
 * @author Yassin Ajdi.
 */
data class TrailersResponse (

    @SerializedName("results")
    var trailers: List<Trailer>? = null
)
